test/miner_blockchain_SUITE.erl

```diff
@@ -22,6 +22,9 @@
 all() -> [
           restart_test,
           dkg_restart_test,
+          validator_test,
```

Comment:
I'm only nitpicking but can we have an entirely separate suite with the three tests here if it's not too much pain? `miner_validator_SUITE` perhaps?

Reply:
Yes, we should have these more broken up, but that's probably out of scope for this right now.

Reply:
since these are redundant now (they're now tested in core), I'm going to delete all but the transition test.